File: src/test/java/br/com/model/classes/NovoItemPedidoTest.java
package br.com.model.classes;

import org.junit.jupiter.api.Test;

import br.com.newObjects.model.classes.NovoItemPedido;
import br.com.newObjects.model.classes.NovoPedido;
import br.com.newObjects.model.classes.NovoProdutoFornecedor;

public class NovoItemPedidoTest {
    @Test
    void testNovaListaItensPedido() {
        System.out.println(NovoItemPedido.novaListaItensPedido(NovoPedido.novoPedidoSimples(),
                NovoProdutoFornecedor.novaListaProdutosFornecedor()));
    }

    @Test
    void testNovoItemPedido() {
        System.out.println(NovoItemPedido.novoItemPedido(NovoPedido.novoPedidoSimples()));
    }
}

File: src/test/java/br/com/model/NovoItemPedidoTest.kt
package br.com.model

import br.com.newObjects.model.NovoItemPedido
import br.com.newObjects.model.NovoPedido
import br.com.newObjects.model.NovoProdutoFornecedor
import org.junit.jupiter.api.Test

class NovoItemPedidoTest {
    @Test
    fun testNovaListaItensPedido() {
        val a = NovoPedido.novoPedido()
        NovoItemPedido.novaListaItensPedido(a, NovoProdutoFornecedor.novaListaProdutosFornecedor())
        println(a!!.itensPedido)
    }

    @Test
    fun testNovoItemPedido() {
        val a = NovoPedido.novoPedido()
        NovoItemPedido.novoItemPedido(a)
        println(a!!.itensPedido)
    }
}
